src/node/rpc/frontend.h

```diff
@@ -434,6 +434,26 @@ namespace ccf
           return jsonrpc::success(out);
         };
 
+      auto who_am_i = [this](const RequestArgs& args) {
+        return jsonrpc::success(WhoAmI::Out{args.caller_id});
+      };
+
+      auto who_is = [this](Store::Tx& tx, const nlohmann::json& params) {
+        const WhoIs::In in = params;
+
+        auto certs_view = tx.get_view(*certs);
```

Comment:
This will dereference a `nullptr` when executed on the nodes frontend. We should check that `certs != nullptr` here and return an error when it is.

Reply:
I've added nullptr guards here and below. It would be neater to just avoid installing these methods `if (certs == nullptr)`, but I don't want to introduce that pattern (complicating the "what methods are available" story) right now. At some point I'd like to enforce this distinction more cleanly, perhaps with a deeper inheritance hierarchy of `Frontend`s